docs/schema.md

```diff
@@ -100,7 +100,8 @@ A JSON Schema for validation is also available
 	} ],
 	"credits": [ {
 		"name": string,
-		"contact": [ string ]
+		"contact": [ string ],
+    "type": [ string ]
```

Comment:
Can you please fix the indentation here to be consistent with the rest of the file?

Reply:
I tried but it seems like GH is hellbent on displaying the diff like that 😭 I even copy/pasted the indentation from previous lines to try to match it and as far as I can tell it's consistent with the rest of the file.